[Workers] webcrypto revision (#3081)
cloudflare/cloudflare-docs: products/workers/src/content/runtime-apis/web-crypto.md

<Aside type="warning" header="Warning">

> The Web Crypto API differs significantly from Node’s Crypto API. If you want to port JavaScript that relies on Node’s Crypto API, you will need to invest in translating it to use Web Crypto primitives.

Reviewer: ```suggestion
The Web Crypto API differs significantly from Node’s Crypto API. If you want to port JavaScript code that relies on Node’s Crypto API, you will need to adapt it to use Web Crypto primitives.
```
`invest in translating it to use` seemed a bit convoluted.

### Supported algorithms

Workers implements all operation of the [WebCrypto standard](https://www.w3.org/TR/WebCryptoAPI), as shown in the following table.

> The Workers team continously adds support for more algorithms — [share your use case with the community](https://community.cloudflare.com/c/developers/workers).

Reviewer: ```suggestion
The Workers team continuously adds support for more algorithms — [share your use case with the community](https://community.cloudflare.com/c/developers/workers).
```

1. <a name="footnote-1"></a> Non-standard EdDSA is supported for the Ed25519 curve. Since this algorithm is non-standard, a few things to keep in mind while using it:
* Use <Code>NODE-ED25519</Code> as the algorithm and namedCurve parameters.
* Unlike NodeJS, Cloudflare will not support raw import of private keys.

> * Since this algorithm is non-standard, the implementation may change over time. While Cloudflare cannot guarantee it at this time, Cloudflare will strive to maintain backward compatability and compatibility with NodeJS's behavior.

Reviewer: ```suggestion
* The algorithm implementation may change over time. While Cloudflare cannot guarantee it at this time, Cloudflare will strive to maintain backward compatibility and compatibility with NodeJS's behavior.
```
We already have `Since this algorithm is non-standard` before the bulleted list.

Any notable compatibility notes will be communicated in release notes and via this developer document.

> 2. <a name="footnote-2"></a> MD5 is not part of the WebCrypto standard but is supported in Cloudflare Workers for interacting with legacy systems that require MD5. MD5 is considered a weak algorithm. Do not rely upon MD5 for security.

Reviewer: ```suggestion
2. <a name="footnote-2"></a> MD5 is not part of the WebCrypto standard but is supported in Cloudflare Workers for interacting with legacy systems that require MD5. MD5 is considered a weak algorithm — do not rely upon MD5 for security.
```
